src/routes/languages/+page.svelte

```diff
@@ -7,26 +7,66 @@
-        <li><a href="/languages/nashan"><strong>Nashan</strong></a> (coming soon), the official language of the Nashan Empire by 2000AR</li>
+        <li><a href="/languages/nashan"><strong>Nashan</strong></a>, the official language of the Nashan Empire by 2000AR</li>
         <li><a href="/languages/pthm"><strong>Pthm</strong></a>, the literary language of the Fifgath by 1600AR</li>
+        <li><a href="/languages/alkan"><strong>Alkan</strong></a>, the language of Alka and Sardiki in the 2000's AR</li>
+        <li><a href="/languages/permechikan"><strong>Permechikan</strong></a>, the language of the Permechikan Empire and its successor states, after 1800 AR</li>
+        <li><a href="/languages/prins"><strong>Prins</strong></a>, the language of the Prins Empire, 2000AR</li>
+        <li><a href="/languages/xomitec"><strong>Xomitec</strong></a>, the language of the Xomitec Empire in the 2000's AR</li>
     </ul>
 
     <h3 class="text-left">Cursed Conlangs, Jokelangs</h3>
+    <p>These are the nightmarish abominations created by Nguh and Ætérnal over the years:
+
+    </p>
     <ul>
         <li><a href="/languages/gumsmaq"><strong>Gumsmaq</strong></a>, the language of eating food</li>
         <li><a href="/languages/hyperpirate"><strong>HyperPirate</strong></a>, the language of pirates</li>
         <li><a href="/languages/rabbid"><strong>Rabbid (Lapin Cretin)</strong></a>, the language of the Rabbids from Ubisoft's Rayman Raving Rabbids series</li>
         <li><a href="/languages/santaa"><strong>Santaa</strong></a>, the language of Santa and Christmas</li>
         <li><a href="/languages/ultrafrench"><strong>ULTRAFRENCH</strong></a>, remember French, that old language from centuries ago? Well this is what came after.</li>
+        <li><a href="/languages/hallowed-ween"><strong>The Hallowed Ween</strong></a>, the language of American Ghosts</li>
+        <li><a href="/languages/hyperformal"><strong>HyperFormal</strong></a>, the language of the HyperFormal, natural enemies of the Hyperpirates</li>
+        <li><a href="/languages/portugoose"><strong>Portugoose</strong></a>, the language of Portugeese</li>
+        <li><a href="/languages/secret-handshake"><strong>Secret Handshake</strong></a>, a minilang for secret communication with one hand</li>
     </ul>
 
-    <h3 class="text-left">Conlangs by other creators showcased on the Agma Schwa Channel</h3>
-    <ul>
-        <li><a href="https://youtu.be/PoUqMjboZLs"><strong>Kantwo</strong></a>, by Kaentwo</li>
-        <li><a href="https://youtu.be/tATcokhBvQo"><strong>Mondir</strong></a>, by Salum the Conlang</li>
-    </ul>
+</section>
+<Stripe>Sern-Cahilan Dictionaries</Stripe>
+<section>
+    <h3 class="text-left">Sern-Cahilan Unified Dictionary with PSC ID#s </h3>
+    <p>The spreadsheet below contains the unified Sern-Cahilan Dictionary that shows how the root words of every applicable word inherited from Proto-Sern-Cahilan (Hvasvan).
```

Comment:
```suggestion
    <p>The spreadsheet below contains the unified Sern-Cahilan Dictionary that shows how the root words of every applicable word inherited from Proto-Sern-Cahilan (Hvasvan).
```
Er, something seems wrong about this sentence (that or it’s just a garden-path sentence for me). Can you give that another read?

Reply:
I'll split that into a couple sentences.